packages/wonder-blocks-core/util/ssr-id-factory.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import {IIdentifierFactory} from "../util/types.js";
+import type {IIdentifierFactory} from "../util/types.js";
```

Comment:
rollup complained about this but webpack didn't which I fond a bit surprising.

Reply:
Huh, fascinating!